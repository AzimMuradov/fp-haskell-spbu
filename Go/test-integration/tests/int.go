func main() {
  var a = 42;
  var b = 17;

  println(+a);     // should print "42"
  println(-a);     // should print "-42"
  println(a == b); // should print "false"
  println(a != b); // should print "true"
  println(a < b);  // should print "false"
  println(a <= b); // should print "false"
  println(a > b);  // should print "true"
  println(a >= b); // should print "true"
  println(a + b);  // should print "59"
  println(a - b);  // should print "25"
  println(a * b);  // should print "714"
  println(a / b);  // should print "2"
  println(a % b);  // should print "8"
}
