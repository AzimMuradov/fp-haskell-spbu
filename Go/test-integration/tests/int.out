42
-42
false
true
false
false
true
true
59
25
714
2
8
